system-tests/README.md

```diff
@@ -19,3 +19,35 @@ It uses Docker containers to create containerised instances of Kafka and other c
 * Wait
 
 Note: these tests take some time to run.
+
+
+### General Architecture
+
+
+The system tests use pytest for the test runner, and use separate fixtures for different configurations of the file-writer. 
+
+Firstly, the system tests attempt to build and tag the latest file-writer image. This can take a lot of time especially if something in conan has changed, as it has to reinstall all of the conan packages.
+
+The Kafka and Zookeeper containers are started with `docker-compose` and persist throughout all of the tests, and when finished will be stopped and removed.
+
+Each fixture starts the file-writer with an `ini` config file (found in `/config-files`), and in some cases use a JSON command at startup so nothing has to be sent over kafka. Some fixtures start the filewriter multiple times and others use the NeXus-streamer image. 
+
+In some tests, command messages in `JSON` form are sent to kafka to change the configuration of the file-writer during testing. 
+
+Most tests check the NeXus file created by the file-writer contain the correct static and streamed data however some tests also consume everything from the status topic to assert against. 
```

Comment:
"...contains..."

Reply:
Suggest rewording the latter half of the sentence:
"...some tests instead test that the status of the file writer matches expectation, by consuming status messages from Kafka."